### PR TITLE
Implements the new Codex.

### PR DESCRIPTION
**Pull request by Neerti**

### CODE REVIEW

File: code/modules/lore_codex/lore_data/important_locations.dm [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	data = "Outside of the habitable zone, the barren world Magni is generally at 202 kelvin (-71�C)."
+
+/datum/lore/codex/page/kara/add_content()
+	name = "Kara (Gas Giant)"
+	keywords += list("Kara")
+	data = "A gas giant, with a large number of moons. Captured asteroids, to be specific. Many of the asteroids are theorized \
+	to be the remnants of a much larger moon that was ripped apart by Kara, long ago. Curerntly, a large number of these \
+	asteroids are being used by many different businesses, and some governmental infrastructure has been built. The most prominent \
+	asteroid installation is the [quick_link("Northern Star", "NCS Northern Star")], a general purpose colony owned and operated by \
+	[quick_link("NanoTrasen")]. The mid-atmospheric temperature of the gas giant is 150 kelvin (-108�C)."
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/important_locations.dm]: Can we toss the word "average" into the temp thing?

File: code/modules/lore_codex/lore_data/important_locations.dm [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	<br><br>\
+	Today it houses a population of civilians, whom work to maintain \
+	the colony and support the local mining industry.  The colony also has managed to achieve a degree of \
+	self-sufficiency, and possesses many amenities and features that most other asteroid bases in the \
+	subsystem lack."
+
+/datum/lore/codex/page/rota/add_content()
+	name = "Rota (Gas Giant)"
+	keywords += list("Rota")
+	data = "An ice giant, with a beautiful ring system circling it. It is 165 kelvin (-157�C)."
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/important_locations.dm]: Can we toss the word "average" into the temp thing?

File: code/modules/lore_codex/lore_data/orgs.dm [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		P.name = O.name
+		P.keywords.Add(O.name, O.short_name)
+		if(O.acronym)
+			P.keywords.Add(O.acronym)
+		P.data = O.desc
+		children.Add(P)
+
+/datum/lore/codex/category/auto_org/tsc
+	name = "Trans-Stellar Corporations"
+	data = "By definition, TSCs are companies which span multiple star systems, however the team is generally reserved for \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/orgs.dm]: Term, not team?

File: code/modules/lore_codex/lore_data/political_parties.dm [hunk 4]
```diff
@@ -0,0 +1,99 @@
+/datum/lore/codex/category/political_factions
+	name = "Political Factions"
+	data = "Those wishing to immigrate to somewhere in Vir, or otherwise plan to stay for a long time should get to know human politics.  \
+	There are presently three major political parties that exist throughout SolGov space, being the Icarus Front, the Shadow Coalition, and \
+	the Sol Economic Organization, and several smaller ones which tend to align themselves among one of the majors.  In the Vir system, the \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/political_parties.dm]: one of the major parties?

File: code/modules/lore_codex/lore_data/political_parties.dm [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+	conservative body with a long history, tracing its linage back to the political unrest that created the Sol Confederate Government. Icarus calls \
+	for severe restrictions on \"transformative technologies\" any technology with the power to fundamentally alter humanity, such as advanced artificial \
+	intelligence and human genetic augmentation. Previously an unbeatable political force, recent changes have lead to its power backsliding. It remains a \
+	popular party among those from Sol, Tau Ceti, and other heavily settled systems."
+
+/datum/lore/codex/page/shadow_coalition/add_content()
+	name = "Shadow Coalition"
+	data = "A disorganized liberal party, originating in an anti-[quick_link("Icarus")] shadow government.  'Shadow' in this case, refers to acting as an opposition \
+	party to the Icarus majority. The Shadow Coalition calls for the lifting of certain Icarus-restricted technologies, especially medical \
+	technologies with the ability to drastically improve quality of life. While fractious and bound to infighting, the Shadow Coalition and affiliated \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/political_parties.dm]: prone to infighting?

File: code/modules/lore_codex/lore_data/political_parties.dm [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+	a measure supported due to its potential to counteract the aging positronic population and to bring the average positronic closer to human culture.  They tend to vote \
+	along with the [quick_link("Shadow Coalition")], due to being Technoprogressive."
+
+/datum/lore/codex/page/church_of_unitarian_god
+	name = "The Church of the Unitarian God"
+	keywords = list("Unitarian Church")
+	data = "An often-imperfect fusion of various human religions such as Christianity, Islam, and Judaism, the Unitarian Church represents the dim voice of \
+	religion in a time of increased atheism. With the threat of singularity looming once more, their power is increasing with more converts and more donations, \
+	and they use this power to protect the fundamental human soul from corruption by dangerous technologies and to spread their faith among aliens and positronics, \
+	who they view as fellow children of God.  They tend to side with Bioconseratives."
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/political_parties.dm]: Bionconservatives, missed a v.

File: code/modules/lore_codex/lore_data/species.dm [hunk 7]
```diff
@@ -0,0 +1,301 @@
+/datum/lore/codex/category/species
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Have you passed these by the Loremasters?

Comment (Neerti) [inline code/modules/lore_codex/lore_data/species.dm]: Most of the text was ripped from the wiki.

File: code/modules/lore_codex/lore_data/species.dm [hunk 8]
```diff
@@ -0,0 +1,301 @@
...
+	outside of Moghes tend to be exiles however, and with influence of other species the gender difference is not nearly as pronounced. Unathi were \
+	humanity's second contact, and despite their aggressive nature, seem to get along well enough with humanity, though are often considered to be \
+	'second-class' citizens and are rarely seen in jobs other than where muscle is needed." // This probably needs to be updated.
+	*/
+
+/datum/lore/codex/page/tajaran
+	name = "Tajaran"
+	keywords = list("Tajara")
+	data = "The Tajara are a race of humanoid mammalian aliens from Meralar, the fourth planet of the Rarkajar star system. Thickly furred and protected \
+	from cold, they thrive on their subartic planet, where the only terran temperate areas spread across the equator and tropical belt. \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Spelling on Subarctic

File: code/modules/lore_codex/lore_data/species.dm [hunk 9]
```diff
@@ -0,0 +1,301 @@
...
+	With their own share of bloody wars and great technological advances, the Tajaran are a proud kind. They fiercely believe they belong \
+	among the stars and consider themselves a rightful interstellar nation, even if the humans helped them to actually achieve superluminar \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Spelling on Superluminal

File: code/modules/lore_codex/lore_data/species.dm [hunk 10]
```diff
@@ -0,0 +1,301 @@
...
+	themselves to training as technicians and engineers in order to obtain skills and training to take back to Sirisai. \
+	The pack is only around thirty people in size, but owns and maintains a nuclear power plant.\
+	<br><br>\
+	<b>Schasaraca</b><br>\
+	One of the more Skrell-devoted and integrated packs. They tend to be rather sycophantic towards the Skrell and work as \
+	scientists and field researchers on a variety of projects, generally biology or technical research. They have a reputation \
+	for working as spies and informants for the Skrell governments amongst other Teshari.\
+	<br><br>\
+	<b>Ceea</b><br>\
+	An isolationistic pack from the northern tundra of Sirisai; generally known as disliking the Skrell. Small to average in size; \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Isolationist, not isolationistic.

File: code/modules/lore_codex/lore_data/species.dm [hunk 11]
```diff
@@ -0,0 +1,301 @@
...
+	side of a wall, but this comes at a cost.  Very loud noises are very painful for Teshari, so be mindful of \
+	your indoor voice when speaking with one. The Teshari are omnivorous but generally prefer to eat meat wherever possible."
+
+// Posi lore
+/datum/lore/codex/category/positronic/add_content()
+	name = "Positronics"
+	keywords += list("Positronic", "Posi", "Posibrain", "Posibrains")
+	data = "A Positronic being, is an individual with a positronic brain, manufactured \
+	and fostered amongst organic life. Positronic brains enjoy the same legal status as a human in [quick_link("SolGov")] space, although discrimination is \
+	still prevelent, and are considered sapient on all accounts. They can be considered the \"synthetic species\". Half-developed and \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Spelling on Prevalent

File: code/modules/lore_codex/lore_data/species.dm [hunk 12]
```diff
@@ -0,0 +1,301 @@
...
+	children = list(
+		/datum/lore/codex/page/positronic_brain_physical,
+		/datum/lore/codex/page/positronic_memory,
+		/datum/lore/codex/page/jans_fhriede
+		)
+
+/datum/lore/codex/page/positronic_brain_physical
+	name = "Physical Structure of a Positronic Brain"
+	keywords = list("Physical Posibrain", "Physical Positronic")
+	data = "A positronic brain is a cube of complex metal alloy between two and sisx inches to a side. They usually weigh just under ten kilograms and are \
```
Comment (Anewbe) [inline code/modules/lore_codex/lore_data/species.dm]: Spelling on six

**Pull request merged**